tests/conformance_ruby/standard_filter_test.rs

```diff
@@ -120,14 +120,13 @@ fn test_escape_once() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#253
 fn test_url_encode() {
     assert_eq!(
         v!("foo%2B1%40example.com"),
         filters!(UrlEncode, v!("foo+1@example.com"))
     );
     assert_eq!(v!("1"), filters!(UrlEncode, v!(1)));
-    assert_eq!(v!("2001-02-03"), filters!(UrlEncode, date(2001, 2, 3)));
+    assert_eq!(v!("2001-02-03"), filters!(UrlEncode, date_string(2001, 2, 3)));
```

Comment:
I think I'd like to keep `date` for now.  Compatibility with liquid-rb will require us to add Date support.

(ditto for decode)

Reply:
How do you want to add Date support? Since every place the repository refers to "Date" it means "DateTime", adding Date will require doing one of two "breaking" things:
- Rename the "Date" concept to "DateTime" and add a "Date" concept. The problem with this is that existing code will be legal but will function differently. The current filter architecture restricts this change, since it maps each parameter type to a single value type, so we can't have a "date" parameter type that starts as a "Date", and then falls back to "DateTime".
- Add a "DateOnly" concept to specify "Date" (vs "DateTime"), which is counter-intuitive.

Reply:
I'd go for renaming `Date` to `DateTime`.  We still have a relatively high-churn API and few users so I'm ok with breaking.